core/core-impl/src/main/java/io/openshift/appdev/missioncontrol/tracking/segment/SegmentAnalyticsProvider.java

```diff
@@ -0,0 +1,102 @@
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Produces;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.messages.MessageBuilder;
+import com.segment.analytics.messages.TrackMessage;
+
+import io.openshift.appdev.missioncontrol.core.api.LaunchEvent;
+import io.openshift.appdev.missioncontrol.core.api.StatusMessage;
+import io.openshift.appdev.missioncontrol.core.api.StatusMessageEvent;
+
+/**
+ * Provides the {@link Analytics} instance to be used
+ *
+ * @author <a href="mailto:alr@redhat.com">Andrew Lee Rubinger</a>
+ * @author <a href="mailto:tschotan@redhat.com">Tako Schotanus</a>
+ */
+@ApplicationScoped
+public class SegmentAnalyticsProvider {
+
+    private static final Logger log = Logger.getLogger(SegmentAnalyticsProvider.class.getName());
+
+    private static final String NAME_EVENT_LAUNCH = "launch";
+    private static final String KEY_OPENSHIFT_PROJECT_NAME = "openshiftProjectName";
+    private static final String KEY_GITHUB_REPO = "githubRepo";
+
+    //TODO Externalize into sysprop
+    private static final String KEY = "oOlNiAf3K5MDwpd4ErD2ZPRe6z3Ckk7w";
+
+    private Analytics analytics;
+
+    @Resource
+    private ManagedExecutorService async;
+
+    public void initAnalytics(@Observes @Initialized(ApplicationScoped.class) Object init) {
+        // TODO Externalize as above
+        final String key = KEY;
+        analytics = Analytics.builder(key).build();
+        log.info("Using Segment analytics with key: " + key);
+	}
+
+    public void onEvent(@Observes LaunchEvent launch) throws IOException {
+        final Callable<Void> task = this.createTrackingTask(
+                launch.getUser(),
+                launch.getId(),
+                launch.getGithubRepo(),
+                launch.getOpenshiftProjectName());
+        async.submit(task);
+	}
+
+    public Callable<Void> createTrackingTask(final String userId,
+                                             final UUID projectileId,
+                                             final String githubRepo,
+                                             final String openshiftProjectName) {
+        return new Callable<Void>() {
+            @Override
+            public Void call() throws Exception {
+                // Create properties
+                final Map<String, String> props = new HashMap<>();
+                props.put(KEY_GITHUB_REPO, githubRepo);
+                props.put(KEY_OPENSHIFT_PROJECT_NAME, openshiftProjectName);
+
+                // Create message
+                final MessageBuilder message = TrackMessage.builder(NAME_EVENT_LAUNCH).
+                        messageId(projectileId).
+                        userId(userId).
+                        properties(props);
+
+                // Send to analytics engine
+                analytics.enqueue(message);
+
+                log.info("Queued tracking message for: " +
```

Comment:
Def all the `INFO` level logging I have in there was for dev only; yep, needs to go to more appropriate level